### ISSUE TITLE
Jay is getting error on project-assemble

### ISSUE THREAD

**Issue by mr-ice** [mr-ice/maptool-macros#70]
```
I also get this message when it does build:
  adding: content.xml (deflated 92%)
  adding: properties.xml (deflated 43%)
rm: can't remove '.temp-23/assets': Directory not empty
make: *** [makefile:30: DND5e.mtprops] Error 1
```

**Comment by mr-ice** [comment]
This may be because the windows filesystem still has files from the temporary directory open when the makefile tries to remove them from within the container.

**Comment by mr-ice** [comment]
Jay said this error was solved by 39ccc31
